json/actions.json

```
@@ -19,6 +19,16 @@
       "slashes": false
     }
   },
+  "issue": {
+    "type": "issue",
+    "data": {
+      "description": "C'è un problema tecnico/tattico, portalo all'attenzione di un mantainer",
+      "responses":[
+        "Mantainer c'è bisogno del vostro aiuto \n",
+        "Scrivete sul gruppo di @csunibo nel topic Bot"
+      ]

```

Come già detto anche in privato non mi piace questo modo di usare gli array per elencare le possibili risposte perché è molto fragile. Tu conosci la risposta giusta soltanto in base all'indice dell'array, che se per caso cambia tutto il codice si rompe. Una soluzione a mio avviso più pulita sarebbe la seguente:
```suggestion
      "response": "Mantainers, c'è bisogno del vostro aiuto \n",
      "fallback": "Nessun mantainer è presente in questo gruppo. Riportate il problema direttamente sul gruppo di @csunibo nel topic Bot"
```
Ha cambiato anche leggermente le risposte per renderle un file più chiare